maven-plugin/src/main/java/io/jenkins/tools/incrementals/maven/IncrementalifyMojo.java

```diff
@@ -146,10 +160,71 @@ public class IncrementalifyMojo extends AbstractVersionsUpdaterMojo {
         prependProperty(pom, "gitHubRepo", connectionRGHR.gitHubRepo);
         prependProperty(pom, "changelist", "-SNAPSHOT");
         prependProperty(pom, "revision", m.group(1));
-        PomHelper.setProjectValue(pom, "/project/scm/tag", "${scmTag}");
-        PomHelper.setProjectValue(pom, "/project/scm/connection", connectionRGHR.interpolableText);
-        PomHelper.setProjectValue(pom, "/project/scm/developerConnection", developerConnectionRGHR.interpolableText);
-        PomHelper.setProjectValue(pom, "/project/scm/url", urlRGHR.interpolableText);
+        PomHelper.setElementValue(pom, "/project/scm", "tag", "${scmTag}");
+        PomHelper.setElementValue(pom, "/project/scm", "connection", connectionRGHR.interpolableText);
+        PomHelper.setElementValue(pom, "/project/scm", "developerConnection", developerConnectionRGHR.interpolableText);
+        PomHelper.setElementValue(pom, "/project/scm", "url", urlRGHR.interpolableText);
+    }
+
+    /**
+     * Gets the parent artifact from the pom.
+     *
+     * @param pom    The pom.
+     * @param helper The helper (used to create the artifact).
+     * @return The parent artifact or <code>null</code> if no parent is specified.
+     * @throws XMLStreamException if something went wrong.
+     */
+    private static Artifact getProjectParent( final ModifiedPomXMLEventReader pom, VersionsHelper helper )
+            throws XMLStreamException
```

Comment:
Copypasta, this was deleted upstream with no explanation. Inlining the last working version of it before it was deleted.